spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/actions/DeleteOrphanFilesSparkAction.java

```diff
@@ -206,6 +208,11 @@ public DeleteOrphanFilesSparkAction compareToFileList(Dataset<Row> files) {
     return this;
   }
 
+  public DeleteOrphanFilesSparkAction usePrefixList(boolean newUsePrefixList) {
```

Comment:
nit: maybe `usePrefixListing`, or add javadoc. I think it is not trivial to understand what this config does